react-app/src/api/useGetSystemNotifs.ts

```diff
@@ -5,8 +5,38 @@ import { BannerNotification, ReactQueryApiError } from "shared-types";
 
 import { useGetUser } from "@/api";
 
+export type Notification = {
```

Comment:
Isn't this the same type as `BannerNotification` in shared-types/notification.ts?

Reply:
Changed